payment-request/allowpaymentrequest/echo-PaymentRequest.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<script>
+window.onmessage = (e) => {
+  const paymentArgs = [[{supportedMethods: ['foo']}], {total: {label: 'label', amount: {currency: 'USD', value: '5.00'}}}];
```

Comment:
Yes, as far as I can tell. It would fail in `show()` though if 'foo' is not an accepted method.